src/mem/port.hh

```diff
@@ -68,37 +71,37 @@ class SlavePort;
  * The three protocols are atomic, timing, and functional, each with its own
  * header file.
  */
-class MasterPort : public Port, public AtomicRequestProtocol,
+class RequestPort: public Port, public AtomicRequestProtocol,
     public TimingRequestProtocol, public FunctionalRequestProtocol
 {
-    friend class SlavePort;
+    friend class ResponsePort;
 
-  private:
-    SlavePort *_slavePort;
+    private:
```

Comment:
Added spaces here. I know it's weird and IDEs complain, but our style says that private/public/protected are *two* spaces indented.

Reply:
Fixed!